rest/accounts/v1/credentials_aws.go

```diff
@@ -152,6 +161,83 @@ func (c *ApiService) ListCredentialAws(params *ListCredentialAwsParams) (*ListCr
 	return ps, err
 }
 
+// Lists CredentialAws records from the API as a list. Unlike stream, this operation is eager and loads 'limit' records into memory before returning.
+func (c *ApiService) ListCredentialAws(params *ListCredentialAwsParams, limit int) ([]AccountsV1CredentialCredentialAws, error) {
```

Comment:
@shwetha-manvinkurke could limit have been added to the Params struct? It should be an optional param

Reply:
Thought we already discussed that. `Params` are auto generated from the specs and are meant to hold the form params and doesn't sound right to include `limit` in it. Yes, limit should be optional and would have been nice if go did allow optional func parameters but this was the cleanest way to achieve that.

Reply:
I think it's worth it to code this into the generator so that the ListParams includes the limit. This makes the signature cleaner.